Dockerfile

```diff
@@ -1,4 +1,4 @@
-FROM golang:1.10.4 as build
+FROM golang:1.11 as build
```

Comment:
Please add:

```
ENV CGO_ENABLED=0
```

Reply:
Will still need this

Reply:
https://github.com/openfaas-incubator/faas-federation/pull/6